java/src/main/c++/vw_learner_VWScalarsLearner.h

```diff
@@ -0,0 +1,30 @@
+/* DO NOT EDIT THIS FILE - it is machine generated */
```

Comment:
We really shouldn't.  This is generated by [javah](http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javah.html).  It just generates the headers.  I think the repo would look pretty weird if it just contained the .cc files.  It might be doable but I don't see the point because if the signature changed on the JNI side that would cause these files to be generated differently then the .cc files would have to change anyway.